modules/extensions/src/views/lite/components/debugger/index.tsx

```diff
@@ -57,20 +57,29 @@ export class Debugger extends React.Component<Props, State> {
     this.props.store.view.removeCustomAction(this.customActionId)
   }
 
-  handleNewMessage = async ({ payload, incomingEventId }) => {
-    if (!['session_reset', 'visit'].includes(payload.type) && incomingEventId !== this.lastMessage) {
-      this.lastMessage = incomingEventId
-      this.loadEvent(incomingEventId)
+  handleNewMessage = async ({ payload, id }) => {
+    if (!['session_reset', 'visit'].includes(payload.type) && id !== this.lastMessage) {
+      this.lastMessage = id
+      await this.loadEvent(id)
     }
   }
 
   handleSelect = async (_actionId: string, props: any) => {
-    this.loadEvent(props.incomingEventId, true)
+    await this.loadEvent(props.id, true)
```

Comment:
It would required handling both cases everywhere. props.id is a message id not an event id.

Besides backward compatibility already works. There is a migration that converts all web_messages to messaging. So the debugger should still work for those old conversations